git_machete/cmd.py

```diff
@@ -3627,7 +3603,7 @@ def check_required_param(in_args: List[str], allowed_values_string: str) -> str:
         else:
             return in_args[0]
 
-    global definition_file_path, up_branch
+    global definition_file_path
```

Comment:
Hmmm can we un-globalize this one?

Reply:
Since we need `definition_file_path` at module level  and manipulate it from functions, the only way to get rid of global statement is to store `definition_file_path` in mutable variable.